data/steam_data/printTypicalSteam.csv

```diff
@@ -0,0 +1,2 @@
+n,filename
+0,printTypicalSteam_0.csv
```

Comment:
Was this file created by you or generated by RAVEN? 
Ideally, source files generated by a human are the majority of what gets saved in a repository. Output files generated by those scripts, unless they were very challenging to generate, do not.